app/mapping/registration/TrustDetailsMapper.scala

```diff
@@ -39,7 +38,7 @@ class TrustDetailsMapper extends Mapping[TrustDetailsType] {
         Logger.info("[TrustDetailsMapper] - Cannot build trust type for Deed of variation yet")
         None
       case (Some(_), None) =>
-        userAnswers.get(KindOfTrustPage).map(mapTrustTypeToDes)
+        ???
```

Comment:
Does this need an implementation or is it handled in settlors now and can it be removed?